mysql-test/suite/tianmu/t/create_db.test

```diff
@@ -0,0 +1,7 @@
+#
+# Test incorrect database names
+#
+--error 1059
+CREATE DATABASE aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa;
+--error 1059
+DROP DATABASE aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa;
```

Comment:
The test case is named create_db, only the incorrect database name is tested. Is the use case incomplete?
Does the incorrect database name also contain some special characters?

Reply:
Yes, this is part of the test, to continue to supplement the completion